docs/configuration/alerting_rules.md

```diff
@@ -79,6 +79,10 @@ groups:
       description: "{{ $labels.instance }} has a median request latency above 1s (current value: {{ $value }}s)"
 ```
 
+Remember that [comparison operators](https://prometheus.io/docs/prometheus/latest/querying/operators/#comparison-binary-operators)
```

Comment:
This page is about alerting rules, not binary operators, so this doesn't belong here.